src/containers/PersonContainer/MattersSponsored.tsx

```diff
@@ -73,9 +74,14 @@ const MattersSponsored: FC<MattersSponsoredProps> = ({ personId }: MattersSponso
           {state.models.map((matterSponsored) => (
             <li key={matterSponsored.id}>
               <dl>
-                <dt>
-                  <strong>{matterSponsored.matter?.name}</strong>
-                </dt>
+                <Link
+                  key={matterSponsored.matter?.id}
+                  to={`/matters/${matterSponsored.matter?.id}`}
+                >
+                  <dt>
+                    <strong>{matterSponsored.matter?.name}</strong>
+                  </dt>
+                </Link>
```

Comment:
Could we pull the `Link` down a few levels so that its immediate child is the `strong` element in line 82? Across pages (except for the "Go to full matter details page" on the event page), the content of a link to the matter page would be just the `matter.name`.